Cargo.toml

```diff
@@ -241,7 +241,7 @@ shader_format_glsl = ["bevy_internal/shader_format_glsl"]
 # Enable support for shaders in SPIR-V
 shader_format_spirv = ["bevy_internal/shader_format_spirv"]
 
-# Enable some limitations to be able to use WebGL2. If not enabled, it will default to WebGPU in Wasm
+# Enable some limitations to be able to use WebGL2. If not enabled, it will default to WebGPU in Wasm. Please refer to the [WebGL2 and WebGPU](../examples/README.md) section of the examples README for more information on how to run Wasm builds with WebGPU.
```

Comment:
this doc is published here: https://docs.rs/bevy/latest/bevy/#default-features
how does that link works from docs.rs?

Reply:
Ah, I didn't know this. Would linking to the file on the `latest` branch work better?

Reply:
yup!

Reply:
Fixed in 11a1caaa0eba83122eda9fa06b332a2ca190ec03